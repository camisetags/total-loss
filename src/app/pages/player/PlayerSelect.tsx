import * as React from 'react';

const PlayerSelect = () => (
  <div>
    <h1 className="main-title">Quem vai jogar?</h1>
    <div className="players-container">
      <div className="side-buttons">
        <button className="button mid side-button button-margin">
          <i className="arrow-icon up" />
        </button>
        <button className="button mid side-button button-margin">
          <i className="arrow-icon down" />
        </button>
      </div>

      <div className="users-cards">
        <div className="card" />
        <div className="card" />
        <div className="card" />
        <div className="card" />
      </div>
      {/* <input type="text" className="inputfield" placeholder="Jogador" /> */}
    </div>
  </div>
);

export default PlayerSelect;
