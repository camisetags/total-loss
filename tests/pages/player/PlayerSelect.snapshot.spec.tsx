import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PlayerSelect from '../../../src/app/pages/player/PlayerSelect';

describe('PlayerSelect page', () => {
  it('should render the PlayerSelect page', () => {
    const rendered = renderer.create(<PlayerSelect />);

    expect(rendered.toJSON).toMatchSnapshot();
  });
});
