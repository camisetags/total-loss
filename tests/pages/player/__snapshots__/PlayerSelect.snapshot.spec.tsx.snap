// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerSelect page should render the PlayerSelect page 1`] = `[Function]`;
